BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkCommentsRequest.m

```diff
@@ -8,6 +8,7 @@
 
 #import "BOXAPIJSONOperation.h"
 #import "BOXComment.h"
+#import "BOXDispatchHelper.h"
```

Comment:
We discussed offline, the base class previously imported this in the .h, even though it wasn't used or needed there. These subclasses who use it should import it (since some may not).